_agencies/agency-international-development.html

```diff
@@ -1,6 +1,5 @@
 ---
 name: Agency for International Development
-data_url: https://s3-us-gov-west-1.amazonaws.com/cg-ff27f612-cd16-4a76-94f8-628986af2158/data/usaid
 slug: agency-international-development
```

Comment:
I also investigated removing the `slug` field from each template file by replacing it with something automatically derived from the filename of the template, but Jekyll didn't provide a sane hook to grab this in a way that would work for both e.g. `/agriculture/` and `/agriculture/data/`.